core/digital-twin-lib/src/DAppContainer.ts

```diff
@@ -298,12 +311,34 @@ class DAppContainer extends Container {
     return plugin;
   }
 
-  async getPermissions(): Promise<ContainerShareConfig> {
-    return {
-      read: [],
-      readWrite: [],
-      ...await this.getContainerShareConfigForAccount(this.runtime.activeAccount),
+  /**
+   * Returns permissions object with the same structure as the plugin.
+   * Adds `type {read: true}` to permissions, because it's omitted in getContainerShareConfigForAccount() function.
+   */
+  async getPermissions(): Promise<Permissions> {
+    const {
+      read = [],
```

Comment:
What does this syntax do? Init a const var with `[]` and then overwrite it? Does that even work?

Reply:
It's destructuring assignment with default var, if the returned 'read' is undefined, it's set to empty array.